src/loaders/configure/configure.js

```diff
@@ -43,13 +46,13 @@ export function configure (agentIdentifier, opts = {}, loaderType, forceDrain) {
     ...(updatedInit.ajax.deny_list || []),
     ...(updatedInit.ajax.block_internal ? internalTrafficList : [])
   ]
-  setRuntime(agentIdentifier, runtime)
-
-  setTopLevelCallers()
-  const api = setAPI(agentIdentifier, forceDrain)
-  gosNREUMInitializedAgents(agentIdentifier, api, 'api')
-  gosNREUMInitializedAgents(agentIdentifier, exposed, 'exposed')
-  addToNREUM('activatedFeatures', activatedFeatures)
+  setRuntime(agent.agentIdentifier, runtime)
 
-  return api
+  if (!alreadySetOnce) {
+    setTopLevelCallers() // no need to set global APIs on newrelic obj more than once
+    addToNREUM('activatedFeatures', activatedFeatures)
+  }
```

Comment:
nitpick but can you just combine this with the other alreadySetOnce if statement on 37?